src/nvapi_d3d11.cpp

```diff
@@ -69,25 +69,27 @@ extern "C" {
         if (pMultiGPUCaps == nullptr)
             return InvalidArgument(n);
 
-        switch (pMultiGPUCaps->version) {
-            case 0: // NV_MULTIGPU_CAPS_V1 has no version field
-            case NV_MULTIGPU_CAPS_VER1: {
+        auto version = pMultiGPUCaps->version;
+        switch (version) { // NOLINT(*-multiway-paths-covered)
+            case NV_MULTIGPU_CAPS_VER2:
+                *pMultiGPUCaps = {};
+                pMultiGPUCaps->version = version;
+                pMultiGPUCaps->multiGPUVersion = 3; // Observed on Windows
+                // Report on purpose that no SLI GPU is available
+                pMultiGPUCaps->nTotalGPUs = nvapiAdapterRegistry->GetAdapterCount();
+                pMultiGPUCaps->nSLIGPUs = 0;
```

Comment:
`*pMultiGPUCaps = {};` implicitly zeroes last two fields, but then `pMultiGPUCaps->nSLIGPUs = 0;` explicitly sets only one of them. Why should we handle this one differently?

Reply:
> `*pMultiGPUCaps = {};` implicitly zeroes all (including last two) fields, but then `pMultiGPUCaps->nSLIGPUs = 0;` explicitly (and redundantly) sets another one. Why should we handle this one differently?

Yeah, it's completely redundant. I think the former me wanted to make it even more obvious that we report zero SLI GPUs when reading the source. Should turn this into a comment.